llvm/lib/Target/NVPTX/NVPTXUtilities.cpp

```diff
@@ -179,6 +179,13 @@ static bool argHasNVVMAnnotation(const Value &Val,
   return false;
 }
 
+static std::optional<unsigned> getFnAttrParsedIntOrNull(const Function &F,
+                                                        StringRef Attr) {
+  if (F.hasFnAttribute(Attr))
+    return F.getFnAttributeAsParsedInteger(Attr);
+  return std::nullopt;
```

Comment:
Nit: Could be just `return F.hasFnAttribute(Attr) ? F.getFnAttributeAsParsedInteger(Attr) : {};`

Reply:
Had to be a little more explicit to make the compiler happy but I've switched to a ternary as requested.